src/streams/TopicItem.js

```diff
@@ -52,15 +53,22 @@ export default function TopicItem(props: Props) {
     unreadStreams: getUnreadStreams(state),
   }));
 
+  const subscription = backgroundData.subscriptions.find(x => x.name === stream);
+
   return (
     <Touchable
       onPress={() => onPress(stream, name)}
       onLongPress={() => {
+        if (!subscription) {
+          logging.error('Stream id does not exist.');
```

Comment:
(same comment as above, about the error message, whether we expect this to happen, and `invariant`)

Reply:
On exploring this I realized that the current implementation allows a user to Long Press `TopicItem` on an unsubscribed stream (from `TopicListScreen`) (I think ideally this shouldn't be the case).
In that scenario subscription will be undefined.
Given that case I am not sure if we should be `logging.error` as well, so I removed that statement in the new revision and now the code simply returns on not finding subscription.